dom.bs

```diff
@@ -1113,6 +1131,12 @@ participate in a tree structure.</p>
  <a for="event listener">callback</a>, and <a for="event listener">capture</a> is
  <var>listener</var>'s <a for="event listener">capture</a>, then <a for=list>append</a>
  <var>listener</var> to <var>eventTarget</var>'s <a>event listener list</a>.
+
+  <li><p>If <a for="event listener">signal</a> is not null then <a for=AbortSignal lt=add>add the following</a>
```

Comment:
Newline before `<p>` (plus indent by a space) as this `<li>` contains multiple children.

Also comma before "then".